src/components/common/footer/index.tsx

```diff
@@ -151,7 +151,7 @@ function Footer() {
         <div className={styles.copyRights}>
           <p>
             Copyright owned by the volunteers contributing to the system and its
-            contents (2006 - 2021)
+            contents (2006 - 2022)
```

Comment:
We should use 
`<p>{new Date().getFullYear()}</p>` to get the current year

Reply:
Done